misc/test-client/Cargo.toml

```diff
@@ -8,10 +8,11 @@ star-constellation = "0.2"
 base64 = "0.21"
 rand = "0.8"
 clap = { version = "4.5", features = ["derive"] }
-reqwest = "0.11"
+reqwest = { version = "0.11", features = ["json"] }
```

Comment:
FWIW v0.12 just came out which migrates to hyper v1

Reply:
gotcha, will upgrade